templates/Web/Pages/Angular.Blank/src/app/app-shell/Param_SourceName_Kebab/Param_SourceName_Kebab-routing.module.ts

```diff
@@ -1,4 +1,4 @@
-import { NgModule } from '@angular/core';
+﻿import { NgModule } from '@angular/core';
```

Comment:
Why are these files marked as changed? Look the same to me? Is it tailing whitespace?

Reply:
Encoding I think, should be UTF-8 with BOM (signature) now. Rather than UTF-8 without BOM.

Reply:
Yes, these are due to encoding changes. These files have been changed to UTF8 + BOM. I have mentioned this in the PR description. :)

Reply:
ups sorry I missed that